pkg/cmd/roachtest/tests/mixed_version_cdc.go

```diff
@@ -401,6 +402,10 @@ func (cmvt *cdcMixedVersionTester) runWorkloadCmd(r *rand.Rand) *roachtestutil.C
 func (cmvt *cdcMixedVersionTester) initWorkload(
 	ctx context.Context, l *logger.Logger, r *rand.Rand, h *mixedversion.Helper,
 ) error {
+	if err := enableTenantSplitScatter(l, r, h); err != nil {
```

Comment:
Seems kinda random here, but I get why it's needed. Maybe the planner can automatically enable this for multi-tenant scenarios?

Reply:
Hrmm, maybe, but some tests work fine without it -- it's only needed for tests that invoke workloads that perform split/scatter. For now I think having tests call this helper function explicitly is fine.